<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacrosDialog</class>
 <widget class="QDialog" name="MacrosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macros</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameToolbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/save.png</normaloff>:/images/48x48/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="shortcut">
         <string notr="true"/>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnOpen">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/open.png</normaloff>:/images/48x48/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnInsertBellow">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/insert_below.png</normaloff>:/images/48x48/insert_below.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnInsertAbove">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/insert_above.png</normaloff>:/images/48x48/insert_above.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRemove">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/remove.png</normaloff>:/images/48x48/remove.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSendAll">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::StrongFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="main.qrc">
          <normaloff>:/images/48x48/send.png</normaloff>:/images/48x48/send.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelProto">
          <property name="text">
           <string>Protocol</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboProto"/>
        </item>
        <item>
         <widget class="QLabel" name="labelDelay">
          <property name="text">
           <string>Delay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinDelay">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkRepeat">
          <property name="text">
           <string>Repeat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>57</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnSave</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>btnInsertBellow</tabstop>
  <tabstop>btnInsertAbove</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnSendAll</tabstop>
  <tabstop>comboProto</tabstop>
  <tabstop>spinDelay</tabstop>
  <tabstop>checkRepeat</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
